app/controllers/projects_controller.rb

```diff
@@ -36,6 +36,22 @@ def show
     commontator_thread_show(@project)
   end
 
+  def project_invite
```

Comment:
move this to project/invitations controller

Reply:
Sure